docs/guide/agent-server-compatibility.asciidoc

```diff
@@ -65,5 +65,5 @@ Below is a chart that outlines the compatibility between different versions of t
 |Agent Version |APM Server Version
 |0.x |6.3-6.4
 |1.x |6.4
-|2.x |>= 6.5
+|2.x, 3.x |>= 6.5
```

Comment:
This is the the change I added for RUM. 
<img width="772" alt="screen shot 2019-01-29 at 2 03 34 pm" src="https://user-images.githubusercontent.com/5618806/51943610-be08e580-23ce-11e9-8462-46c8c773318c.png">